src/wasm.rs

```diff
@@ -206,10 +234,17 @@ where
 }
 
 impl<ExecC, QueryC> WasmKeeper<ExecC, QueryC> {
-    pub fn store_code(&mut self, code: Box<dyn Contract<ExecC, QueryC>>) -> usize {
-        let idx = self.codes.len() + 1;
-        self.codes.insert(idx, code);
-        idx
+    /// Stores contract code in the in-memory lookup table.
+    /// Returns an identifier of the stored contract code.
+    pub fn store_code(
+        &mut self,
+        creator: Addr,
+        code: Box<dyn Contract<ExecC, QueryC>>,
+    ) -> AnyResult<u64> {
```

Comment:
Why does this return a `Result` now? It never errors

Reply:
That's right, I have reverted it to `u64`.